command/task_create.go

```diff
@@ -197,3 +205,217 @@ func (c *taskCreateCommand) Run(args []string) int {
+		if tc.Condition != nil {
+			switch tc.Condition.(type) {
+			case *config.ServicesConditionConfig:
+				action := generateServicesConditionBlockAction(tc.Services)
+				ui.Output(generateServiceFieldMsg(action))
+			case *config.CatalogServicesConditionConfig:
+				action := generateServiceModuleInputBlockAction(tc.Services)
+				ui.Output(generateServiceFieldMsg(action))
+			case *config.ConsulKVConditionConfig:
+				action := generateServiceModuleInputBlockAction(tc.Services)
+				ui.Output(generateServiceFieldMsg(action))
+			case *config.ScheduleConditionConfig:
+				action := generateServicesAction(tc.Services)
+				ui.Output(generateServiceFieldMsg(action))
+			}
+		} else {
+			action := generateServicesAction(tc.Services)
+			ui.Output(generateServiceFieldMsg(action))
+		}
+	}
+
+	if isError {
+		return errors.New("invalid config")
+	}
+	return nil
+}
+
+func generateServicesConditionBlockAction(services []string) string {
+	list := `"` + strings.Join(services, `","`) + `"`
+	return fmt.Sprintf(servicesConditionBlockAction, list, list)
+}
+
+const servicesConditionBlockAction = `Please add the suggested names field to your current 'condition "services"' ` +
+	`block and assure that 'use_as_module_input' is true or not present
+
+Suggested replacement:
+|    task {
+|  -   services =  [%s]
+|      condition "services"{
+|  +     names = [%s]
+|  +     use_as_module_input = true
+|        ...
+|      }
+|      ...
+|    }
+`
+
+func generateServicesAction(services []string) string {
+	list := `"` + strings.Join(services, `","`) + `"`
+	return fmt.Sprintf(servicesAction, list)
+}
+
+const servicesAction = `Replace the 'services' field with the following 'condition "services"' block
+
+task {
+  ...
+  condition "services" {
+    names=[%s]
+  }
+  ...
+}`
+
+func generateServiceModuleInputBlockAction(services []string) string {
+	list := `"` + strings.Join(services, `","`) + `"`
+	return fmt.Sprintf(servicesModuleInputBlockAction, list)
+}
+
+const servicesModuleInputBlockAction = `Replace the 'services' field with the following 'module_input' block
+
+task {
+  ...
+  module_input "services" {
+    names=[%s]
+  }
+  ...
+}`
+
+func generateServiceFieldMsg(action string) string {
+	return fmt.Sprintf(servicesFieldMsg, action)
+}
+
+const servicesFieldMsg = `the 'services' field in the task block is not supported ` +
+	`
+
+'services' in a task configuration are to be replaced with the following:
+ * condition "services": if there is _no_ preexisting condition block configured in your task
+ * module_input "services": if there is a preexisting condition block configured in your task
+
+%s
+
+For more details and additional examples, please see:
+https://consul.io/docs/nia/release-notes/0-5-0#deprecate-services-field
+`
+
+func generateSourceFieldMsg(module string) string {
+	return fmt.Sprintf(sourceFieldMsg, module, module)
+}
+
+const sourceFieldMsg = `the 'source' field in the task block is not supported ` +
```

Comment:
@wilkermichael same comment as the above... for all mentions of this phrase, I would suggest on changing the language to provide context to users that may have been used to using 'source'. The only change here I would suggest is to replace "not" with "no longer" so the error message would read like the below:

`the 'source' field in the task block is no longer supported`